packages/remark-stringify/types/test.ts

```diff
@@ -8,6 +8,7 @@ const inferredStringifyOptions = {
   incrementListMarker: false
 }
 
+unified().use(remarkStringify)
```

Comment:
This test is currently failing,

Initially I thought that the extra `Compiler` property on the `Attacher` might be the issue
```ts
interface Stringify extends Attacher<[Partial<RemarkStringifyOptions>]> {
  Compiler: Compiler
}
```

but the tests in https://github.com/unifiedjs/unified/pull/67 passing seem to indicate otherwise.

Other ideas? :thought_balloon:

Reply:
same findings as https://github.com/remarkjs/remark/pull/434#discussion_r310361481